clear_cache.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python

```

Comment:
Good idea to have a script to clear cache... this was an issue while developing the app.